servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/submitPledge.js

```diff
@@ -346,6 +350,7 @@ module.exports = async (_, args, context) => {
 
       const pko = packageOptions.find((pko) => pko.id === plo.templateId)
       plo.vat = pko.vat
+      plo.potPledgeOptionId = pko.potPledgeOptionId
```

Comment:
`accessGranted` is currently missing on the pledge option:
```suggestion
      plo.potPledgeOptionId = pko.potPledgeOptionId
      plo.accessGranted = pko.accessGranted
```